desktop/src/app/components/navbar/warnings/modals/delete-outliers-modal.component.ts

```diff
@@ -33,13 +36,37 @@ export class DeleteOutliersModalComponent {
 
     public cancel = () => this.activeModal.dismiss('cancel');
 
-
+    
     public async onKeyDown(event: KeyboardEvent) {
 
         if (event.key === 'Escape') this.activeModal.dismiss('cancel');
     }
 
 
+    public async initialize() {
+
+        this.affectedDocuments = [];
+
+        const foundDocuments: Array<Document> = (await this.datastore.find({
+            constraints: { ['outlierValues:contain']: this.outlierValue }
+        }, { includeResourcesWithoutValidParent: true })).documents;
+
+        for (let document of foundDocuments) {
+            const category: CategoryForm = this.projectConfiguration.getCategory(document.resource.category);
+            const affectedDocument: AffectedDocument = { document: document, fields: [] };
+
+            for (let fieldName of Object.keys(document.warnings.outliers.fields)) {
+                const field: Field = CategoryForm.getField(category, fieldName);
+                if (!this.hasOutlierValue(document, field)) continue;
+                affectedDocument.fields.push(field);
+            }
+
+            if(affectedDocument.fields.length !== 0) this.affectedDocuments.push(affectedDocument);
```

Comment:
Code style: A space should be added between "if" and the opening bracket.

Reply:
"if(affectedDocument.fields.length !== 0)" can be written as:

if (affectedDocument.fields.length)